iota-core/Cargo.toml

```diff
@@ -0,0 +1,16 @@
+[package]
+name = "iota-core"
+version = "0.1.0-alpha.1"
+authors = ["YuWeiWu <wusyong9104@gmail.com>"]
+edition = "2018"
+
+[lib]
+name = "iota"
+
+[dependencies]
+bee-bundle = { git = "https://github.com/Alex6323/bee-p" }
```

Comment:
since iota-client depends on bee, does iota-core still need to add bee in dependency? same as the iota-conversion package.

Reply:
I want to give a horizontal view in core crate. Make it more like a workspace structure.